#! /bin/sh /usr/share/dpatch/dpatch-run
## 80_security_tools.dpatch by Mike Hommey <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: it seems mismatched in fact it shoud enable system zlib 

@DPATCH@

--- nss/mozilla/security/coreconf/Linux.mk
+++ nss/mozilla/security/coreconf/Linux.mk
@@ -176,3 +176,6 @@
 # Always set CPU_TAG on Linux, OpenVMS, WINCE.
 #
 CPU_TAG = _$(CPU_ARCH)
+
+USE_SYSTEM_ZLIB = 1
+ZLIB_LIBS = -lz
